### ISSUE TITLE
Alternate donation methods

### ISSUE THREAD

**Issue by fraktar** [thredded/thredded#555]
I'd like to contribute via BTC, Skrill or Neteller if possible. 

Keep up the great work <3

**Comment by jayroh** [comment]
Not familiar with skrill or neteller and is BTC bitcoin? We'll look into them @fraktar. In the meantime the best way to support us outside of $ is to TELL people! Tweet about the project! Blog about it! Anything!

@glebm we should talk about some way to divert potential funds like this to some sort of escrow. Right now this would be going to me and I don't think that's quite right.

**Comment by glebm** [comment]
> Not familiar with skrill or neteller and is BTC bitcoin? We'll look into them @fraktar

Skrill is the rebranding of moneybookers, a super-old online wallet system, kinda like paypal. Had a few serious breaches in the past.

BTC is easy to set up with https://blockchain.info/ but not sure how to cash out legally.

> we should talk about some way to divert potential funds like this to some sort of escrow.

Do you know of one? If not, I guess we'd need to form a company and get a shared bank account. Forming one here is easy but accounting is a drag.

/cc @timdiggins

**Comment by timdiggins** [comment]
Yes - it's surprising how expensive it is to do accounts properly (my UK accountant is a bit cheaper @glebm IIRC but similar sort of ball park). While escrow/incorporation sounds like a good plan, unless/until it's really significant money it might be better to just divert through one person who is already accounting for things. I have no problems with that being one of you two @glebm @jayroh. 

As a side note, shame there's not some kind of meta-platform for OSS finance (like pledgie + accounting-software + some kind of escrow) . But I think that would end up looking like money-laundering to the authorities and the compliance overheads would be astonishing I would guess. Idle thoughts anyway.

**Comment by moondcr** [comment]
@glebm Hope you also take a look about [Decred](https://decred.org). :smile:

**Comment by glebm** [comment]
@moondcr I've set up a Decred wallet! DsaDygBi8anrXbXznvritWcKowcMPCFiJeq

**Comment by moondcr** [comment]
@glebm Awesome!  [59de200cd2111b514edbc458ecb172d30762b44f0b13268632fb664f92f14ebf](https://mainnet.decred.org/tx/59de200cd2111b514edbc458ecb172d30762b44f0b13268632fb664f92f14ebf)

**Comment by glebm** [comment]
@moondcr Thank you!
